Add clarifications with regards to brackets and EXPRESSIONS (#288)
mapserver/docs: en/mapfile/expressions.txt

Logical expressions
...............................................................................

> Logical expressions take logical values as their input and return logical values. A logical expression is either 'true'

Reviewer: ```suggestion
Logical expressions take logical values as their input and return logical
values. A logical expression is either 'true' or 'false'.
```

> or 'false'. The full expression needs to be surrounded by brackets, but individual logical expressions only

Reviewer: ```suggestion
The full expression needs to be surrounded by brackets, but individual
logical expressions only require brackets to establish precedence or for clarity.
```

> require brackets to establish precedence or for clarity. 

Reviewer: ```suggestion
```
